CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 # the repo. Unless a later match takes precedence,
 # mentioned account names will be requested for
 # review when someone opens a pull request.
-* @m-dobler @CristinaHG @saeltz @timkersch @maksym-moia @oswaldo @houcros
+* @m-dobler @CristinaHG @saeltz @timkersch @oswaldo @houcros
```

Comment:
Says it contains errors, is Oswaldo part of moia-oss?

Reply:
He is not.